packages/nuxt/src/inspector.ts

```diff
@@ -0,0 +1,20 @@
+import { useLogger, useNuxt } from '@nuxt/kit'
+import { withTrailingSlash, withoutTrailingSlash } from 'ufo'
+import * as chalk from 'chalk'
+
+export function addInspector() {
+  const nuxt = useNuxt()
+  const inspectorRoute = '/__unocss'
+
+  if (nuxt.options.dev && nuxt.options.builder === '@nuxt/vite-builder') {
+    nuxt.hook('listen', (_, listener) => {
+      const logger = useLogger()
+
+      const viewerUrl = `${withoutTrailingSlash(listener.url)}${inspectorRoute}`
+
+      logger.info(`UnoCSS inspector available at: ${
```

Comment:
I am not so sure about if the log is necessary.

Reply:
I thought it would be cool to do that since Tailwind module does something very similar?
That also makes it very easy to check upon the inspector from the console where process is running